app/helpers/additional_details/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module AdditionalDetails::ApplicationHelper
+
+  def convert_time(time)
+    modifier = ''
+    ((time.first == '0') || (time[0..1] == '10') || (time[0..1] == '11')) ? modifier = ' AM' : modifier = ' PM'
+
+    DateTime.strptime(time.gsub(':', ''), '%H%M').strftime('%l:%M') + modifier
```

Comment:
Can you explain what the modifier is doing? I'm a bit confused about the '0' || '10' || '11' part